python/ray/train/torch.py

```diff
@@ -132,6 +137,25 @@ def with_sampler(loader):
                 # SequentialSampler.
                 shuffle = not isinstance(loader.sampler, SequentialSampler)
 
+                def seed_worker(worker_id):
```

Comment:
doesn't seem like worker_id is being used? Does it need to take in the arg?

Reply:
Fixed